src/rest.cpp

```diff
@@ -391,12 +393,94 @@ static bool rest_tx(HTTPRequest* req, const std::string& strURIPart)
     }
 
     default: {
-        return RESTERR(req, HTTP_NOT_FOUND, "output format not found (available: " + AvailableDataFormatsString() + ")");
+        return RESTERR(req, HTTP_NOT_FOUND, "Output format not found (available: " + AvailableDataFormatsString() + ")");
+    }
+    }
+    // not reached
+    return true;
+}
+
+
+static bool rest_tx_sendrawtx(HTTPRequest* req, const std::string& strURIPart)
+{
+    if (!CheckWarmup(req))
+        return false;
+    std::string hashStr;
+    const RetFormat rf = ParseDataFormat(hashStr, strURIPart);
+
+    std::string strBody = req->ReadBody();
+    if (std::string::npos == strBody.find("transaction="))
```

Comment:
It seems you're expecting: `transaction=<binary>`, that seems very odd.
It should just be `<binary>`